src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string
+ * with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values
+ * of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map(style => style.split(':'))
+    .filter(style => style.length === 2);
+
+  return styles.reduce((acc, style) => {
+    acc[style[0].trim()] = style[1].trim();
```

Comment:
try [array desctructuring](https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/)